src/main/java/com/cloudbees/plugins/deployer/DeployEvent.java

```diff
@@ -28,15 +28,14 @@
 import hudson.model.AbstractBuild;
 import hudson.model.Cause;
 
-import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
 /**
  * @author stephenc
  */
-public class DeployEvent implements Serializable {
+public class DeployEvent {
```

Comment:
Yeah if it is not actually serializable, then removing the interface does not break anything that was not already broken.